packages/api/src/brevo-contact/brevo-contact.resolver.ts

```diff
@@ -93,10 +93,11 @@ export function createBrevoContactResolver({
         }
 
         @Query(() => PaginatedBrevoContacts)
-        async manuallyAssignedBrevoContacts(
-            @Args() { offset, limit, email, targetGroupId }: ManuallyAssignedBrevoContactsArgs,
+        async assignedBrevoContacts(
+            @Args() { offset, limit, email, targetGroupId, brevoId }: AssignedBrevoContactsArgs,
         ): Promise<PaginatedBrevoContacts> {
             const targetGroup = await this.targetGroupRepository.findOneOrFail({ id: targetGroupId });
+            const brevoListId = brevoId ? targetGroup.brevoId : targetGroup.assignedContactsTargetGroupBrevoId;
```

Comment:
I do not think it is a good idea to send the brevoId and the targetGroup id, especially since you are not using the brevo id here eitherway. I would suggest to add a boolean maybe 'onlyManuallyAssigned' and use this variable here.